src/System/Commands/DBUpdater/CronJobCommand.php

```diff
@@ -131,6 +131,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       $output
     );
 
+    // Custom translation achievements
+
+    $this->runCronJob(
+      'Retroactively unlock custom translation achievements',
+      ['bin/console', 'catrobat:workflow:achievement:translation'],
+      ['timeout' => self::ONE_DAY_IN_SECONDS],
+      '1 month',
```

Comment:
My original plan was to have this command as a one time task and get deleted once it's done. But I guess it wouldn't hurt to run this once in a while to fix inconsistencies.